README.rst

```diff
@@ -1,86 +1,57 @@
-=============
-eox_tagging
-=============
-
-Eox-tagging (A.K.A. Edunext Open extensions) is an `openedx plugin`_, that adds the capability
+===========
+EOX Tagging
+===========
+|Maintainance Badge| |Test Badge| |PyPI Badge|
+
+.. |Maintainance Badge| image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+   :alt: Maintainance Status
+.. |Test Badge| image:: https://img.shields.io/github/actions/workflow/status/edunext/eox-tagging/.github%2Fworkflows%2Ftests.yml?label=Test
+   :alt: GitHub Actions Workflow Test Status
+.. |PyPI Badge| image:: https://img.shields.io/pypi/v/eox-tagging?label=PyPI
+   :alt: PyPI - Version
+
+Eox-tagging is an `openedx plugin`_, part of the Edunext Open Extensions (aka EOX), that adds the capability
 to tag `edx-platform`_ objects. These tags can be used to categorize, include extra information, and so on.
 
 Installation
 ============
 
-Open edX devstack
-------------------
-
-- Install a supported version of the `Open edX devstack`_. See versions for more details.
-
-- Clone the git repo:
-
-.. code-block:: bash
-
-  cd ~/Documents/eoxstack/src/  # Assuming that devstack is in  ~/Documents/eoxstack/devstack/
-  sudo mkdir edxapp
-  cd edxapp
-  git clone git@github.com:eduNEXT/eox-tagging.git
-
-- Install plugin from your server (in this case the devstack docker lms shell):
-
-.. code-block:: bash
-
-  cd ~/Documents/eoxstack/devstack  # Change for your devstack path (if you are using devstack)
-  make lms-shell  # Enter the devstack machine (or server where lms process lives)
-  cd /edx/src/edxapp/eox-tagging
-  pip install -e .
-  python manage.py lms migrate eox_tagging --settings=<your app settings>
-
-Open edX with Tutor (Maple)
-----------------------------
-If you are using `Tutor <https://docs.tutor.overhang.io/gettingstarted.html>`_ to deploy the Open edX instance, please follow this steps:
+#. Install a new fresh instance of tutor following `this steps <https://docs.tutor.overhang.io/quickstart.html#quickstart-1-click-install>`_. *If your instance is running, you can skip this step.*
+#. Add to the Tutor configuration in the file ``cat "$(tutor config printroot)/config.yml"`` these lines that install eox-tagging and eox-core lib:
```

Comment:
If eox-tagging depends on eox-core, why isn't eox-core a package dependency? Here:  https://github.com/eduNEXT/eox-tagging/blob/master/setup.py#L65-L94. I think at some point we should add:
https://github.com/eduNEXT/eox-core/blob/master/setup.py#L73
So eox-core is installed as a dependency, then we can drop this manual install.

Reply:
@dcoa: please, help me add this somewhere so we can revisit it later.

Reply:
I will create an issue to track it.

Reply:
I think we're missing adding this line [here](https://github.com/eduNEXT/eox-tagging/blob/master/setup.py): https://github.com/eduNEXT/eox-core/blob/master/setup.py#L73
So it gets installed. You can test the before and after to make sure.

Reply:
It's not missing https://github.com/eduNEXT/eox-tagging/blob/master/setup.py#L87